llvm/include/llvm/ObjCopy/CommonConfig.h

```diff
@@ -245,6 +245,11 @@ struct CommonConfig {
   StringMap<uint64_t> SetSectionType;
   StringMap<StringRef> SymbolsToRename;
 
+  // Adjust section VMA and LMA
+  int64_t SectionVMA = 0;
+  StringRef SectionName;
+  bool SectionSet = false;
```

Comment:
```suggestion
  // Adjust section VMA and LMA.
  int64_t SectionVMA = 0;
  StringRef SectionName;
  bool SectionSet = false;
```
Could you be more precise with these variable names, please, since the `SectionName` and `SectionSet` variables could be for anything, and the `SectionVMA` variable should probably be named more after the corresponding option? Also, leverage `std::optional` rather than a boolean property, or even just treat an empty `SectionName` as unset, depending on how it's used.